README.md

```diff
@@ -1,7 +1,16 @@
+[content-schemas]: https://github.com/alphagov/govuk-content-schemas
+
 # Content Publisher
 
 A unified publishing application for content on GOV.UK
 
+## Nomenclature
+
+  * Content - Some text ([and related fields][content-schemas]) a user wants to publish
```

Comment:
What about adding images and attachements

Reply:
I think we should stick to text+related fields, unless we're planning to incorporate images/attachments into our revisions/editions. This is a good question - we should be consistent with the word we use to define our other definitions ;-)

Reply:
Also at the moment we don't know how we're going to handle images/attachments so lets leave it out for now